frontend/apps/reader/modules/readerpaging.lua

```diff
@@ -575,6 +575,7 @@ function ReaderPaging:onGotoViewRel(diff)
     else
         self:onGotoPageRel(diff)
     end
+    self.ui:handleEvent(Event:new("PageChangeAnimation", diff > 0))
```

Comment:
What are the reasons to call it via event and not just `self.view:onPageChangeAnimation()`?